From 3e308d91912ade69f914383fd9c13f934da37be4 Mon Sep 17 00:00:00 2001
From: Nagaraju Mekala <nmekala@xilinx.com>
Date: Tue, 20 Apr 2021 21:22:06 +0530
Subject: [PATCH 28/34] [Patch, microblaze]:Changing the long to long long as
 in Windows long is 32-bit but we need the variable to be 64-bit

Signed-off-by :Nagaraju Mekala <nmekala@xilix.com>

Conflicts:
	gas/config/tc-microblaze.c
---
 gas/config/tc-microblaze.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gas/config/tc-microblaze.c b/gas/config/tc-microblaze.c
index 85509011f8c..31677446114 100644
--- a/gas/config/tc-microblaze.c
+++ b/gas/config/tc-microblaze.c
@@ -1017,7 +1017,7 @@ md_assemble (char * str)
   unsigned reg2;
   unsigned reg3;
   unsigned isize;
-  unsigned long immed = 0, immed2 = 0, temp;
+  unsigned long long immed = 0, immed2 = 0, temp;
   expressionS exp,exp1;
   char name[20];
   long immedl;
-- 
2.37.1 (Apple Git-137.1)

